app/models/game.rb

```diff
@@ -1,2 +1,5 @@
 class Game < ApplicationRecord
+  def self.top_scores_by_location(geo_scope, user_location)
+    Game.where("#{geo_scope} = '#{user_location}'").order(score: :desc).limit(5)
+  end
 end
```

Comment:
smart thinking to just return one set of games! love the way this was implemented